### PR TITLE
Container lab static lag fix

### PR DESCRIPTION
**Pull request by indraniBh**

### CODE REVIEW

File: clab/ixia-c-b2b-lag/otg.yaml [hunk 1]
```diff
@@ -91,11 +91,11 @@ layer1:
   promiscuous: true
   speed: speed_1_gbps
 ports:
-- location: te1:5555
+- location: te1:5555;1
```
Comment (apratimmukherjee) [inline clab/ixia-c-b2b-lag/otg.yaml:94]: Can we also add values to dest port to get the traffic to actually hash between tx ports.

**Review by apratimmukherjee — APPROVED** [review]
Please also variate the dest IP

**Comment by apratimmukherjee** [comment]
@bortok : Might be good to also add expectation from the test in the description of the test maybe or as a log when user runs the test . Assuming it is not so easy to actually validate that packets are  hashed across ports and tx=rx in this example.

File: clab/ixia-c-b2b-lag/otg.yaml (hunk 1 above)
Comment (indraniBh) [reply clab/ixia-c-b2b-lag/otg.yaml:94]: Done

**Comment by bortok** [comment]
@apratimmukherjee if you could updated readme on the expectations subject, that would be great

**Comment by apratimmukherjee** [comment]
> @apratimmukherjee if you could updated readme on the expectations subject, that would be great

I have attempted to add the expectations for flow and port stats in the README based on observations when debugging the issue with Indrani. However, I have not set it up and executed it myself to confirm that this is all working with final set of changes.

**Review by bortok — APPROVED** [review]

**Pull request merged**